DevoxxClientMobile/src/main/java/com/devoxx/views/helper/Util.java

```diff
@@ -199,4 +207,38 @@ public static void showToast(String message, Duration duration)  {
     public static String safeStr(String s) {
         return s == null? "": s.trim();
     }
+
+    public static void hidePastConferenceMessage() {
+        if (pastConferenceMessage != null) {
+            pastConferenceMessage.hide();
+        }
+    }
+
+    public static void showPastConferenceMessage() {
+        if (pastConferenceMessage == null) {
+            Label label = new Label(DevoxxBundle.getString("OTN.CONF.PAST.MESSAGE"));
+            Button button = MaterialDesignIcon.CLOSE.button();
```

Comment:
The round region on hover is taller than the box height, should be smaller.

Reply:
Glad that you mentioned. I was actually planning to remove the hover style.